### DISCUSSION TITLE
βίντεο κουϊζ και αποθήκευση απαντήσεων-προόδου

### DISCUSSION THREAD

**Discussion by p18kana** [courses-ionio/hci#1562] [Announcements]
Μπορουμε να βλεπουμε πρωτα ολοκληρο το βιντεο χωρις να απαντησουμε στο youtube και μετα να παμε στις ερωτησεις; Επιπλεον αν κλεισουμε το βιντεο ενω εχουμε απαντησει μια ερωτηση αποθηκευεται η απαντηση η αν μπουμε θα πρεπει να το δουμε απο την αρχη.

**Comment by epidrome** [comment]
όλα τα βίντεο είναι διαθέσιμα δημόσια στου γιουτιουμπ, μπορείτε να τα δείτε όσο θέλετε

το εντπαζλ αποθηκεύει τις απαντήσεις και την πρόοδο σας αν έχετε κάνει λογαριασμό
